### PR TITLE
Add button to toggle between dark/light mode

### PR DESCRIPTION
**Pull request by jishaal**
@finnclark this is what I was thinking for the dark mode toggle btn. Feedback would be primo. 

Lets hope the preview builds 😅

### CODE REVIEW

**Comment by finnclark** [comment]
Hey mate, appreciate the effort but if it's ok, I would be keen to think about this more... Icon toggle buttons aren't a great UX pattern as there's confusion between whether the icon represents the state it's in vs what typically a button does which is describe the action it takes

**Comment by jishaal** [comment]
Sure thing @finnclark 👍🏼

File: app/components/DarkModeToggle/DarkModeToggle.module.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.toggleButton {
+  width: 1.25rem;
+  height: 1.25rem;
```
Comment (finnclark) [inline app/components/DarkModeToggle/DarkModeToggle.module.scss:3]:
```suggestion
  width: 2.5rem;
  height: 2.5rem;
  padding: .625rem;
```

**Review by finnclark — CHANGES_REQUESTED** [review]
Suggestion to increase the touch target to 40px.

another ask - could we see what it's like with the icons swapped so the icon represents the action rather than the state?

**Review by finnclark — APPROVED** [review]
LGTM for now! Might look at something a little different with new design... I'm personally more a fan of a control that's a bit clearer. But this is a great thing to provide value incrementally. Perfection is the enemy of done or something?

**Review by andycarrell — APPROVED** [review]

**Comment by jishaal** [comment]
Thanks @finnclark , fair enough definitely understand this isn't ideal => we can move to the toggle switch or something more clearer once you've had a chance to explore

**Pull request merged**